### ISSUE TITLE
Concierge: Support "admin-free" mode

### ISSUE THREAD

**Issue by browndav** [medic/medic-webapp#1889]
After #1888 is complete, modify Medic OS and `concierge` so that users are created appropriately on first run. This will likely involve (a) switching the full-database administrative user over to a randomly-generated password, and creating the new database-only administrative user required by `medic-webapp`.

**Comment by SCdF** [comment]
This requires a little coordination with #1888 to make sure we meet in the middle.

Here's what I suggest for this ticket:
 - Concierge should modify the `_users` DB to have the `medic-admin` role as an `admin` in `_users`. I.e., if you looked in Futon it would look like this:
![image](https://cloud.githubusercontent.com/assets/583851/12927438/218e6a9c-cf60-11e5-9ade-2d7627ddb068.png)
 - The admin user Concierge creates should use have this role.

_(alternatives would be a different role name, or not using roles and specifically putting the couchdb admin users name in the admin permission in `_users`)_

**Comment by garethbowen** [comment]
Clearly not a priority. Descheduling.

**Comment by abbyad** [comment]
@browndav I am closing as concierge's functionality is soon being replaced by horti. Please re-open and add comment if this issue is still required.

**Comment by browndav** [comment]
Confirming closure: this component is being replaced entirely.
